build/README.md

```diff
@@ -0,0 +1,19 @@
+# Lisk Build
+
+## Linux
+
+A Ubuntu Linux 16.04 LTS (base install) machine is assumed.
+
+### Prerequisistes
+
+The following packages must be installed to build releases with lisk-build:
+
+```
+sudo apt-get install autoconf build-essential git jq libtool moreutils python2.7 tcl8.5 wget zlib1g-dev
+```
+
+### How-To
+
+```
+make LISK_NET=main
```

Comment:
I think better shell syntax is to put ENV variables before start of command. 

```
LISK_NETWORK=mainnet  make
```

Reply:
That is how you pass a variable to make.